lib/topological_inventory/mock_source/collector.rb

```diff
@@ -227,6 +196,11 @@ def targeted_refresh(events)
         save_inventory(parser.collections.values)
       end
 
+      # Used for Save & Sweep Inventory
+      def inventory_name
```

Comment:
hm, this is kind of hidden interface, I might prefer we send it as argument to `save_inventory`